client/src/providers/Context.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useContext, createContext, useEffect } from "react";
+import axios from "axios";
+import PropTypes from "prop-types";
+
+// CREATE USER CONTEXT
+export const UserContext = createContext();
+
+//CUSTOM HOOKS
+// Get Auth
+export const useAuth = () => {
+  // Usage:
+  // const authenticated = useAuth();
+  const authenticated = useContext(UserContext).authenticated;
+  return authenticated;
+};
+// Set Auth
+export const useSetAuthenticated = () => {
+  // Usage:
+  // const setAuthenticated = useSetAuthenticated();
+  // setAuthenticated(true) OR setAuthenticated(false);
+  const setAuthenticated = useContext(UserContext).setAuthenticated;
+  return setAuthenticated;
+};
+// Get User Data
+export const useUserContext = () => {
```

Comment:
I think we should name this `useUserData` instead. 
In the comment let's also briefly outline what the data would include. Something like this:
```
// Usage: 
// const userData = useUserData(); 
// {_id, email, name, googleCredentials, photoUrl, URL, stripeId, subscribed, subscriptionId, ...}
```

which brings me to the question regarding how much of this data we want to expose to the frontend 🧐

Reply:
That makes sense. I can make the name change.